pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -593,11 +603,14 @@ func getAnnotationCheck() error {
 // reconcileForDelete
 func (r *ReconcileChaosEngine) reconcileForDelete(request reconcile.Request) (reconcile.Result, error) {
 
-	r.recorder.Eventf(engine.Instance, corev1.EventTypeNormal, "Stopping reconcile for chaosEngine", "Will remove all Chaos resources for chaosUID: %v", string(engine.Instance.UID))
 	reconcileResult, err := r.removeChaosResources(engine, request)
 	if err != nil {
 		return reconcileResult, err
 	}
+	reconcileResult, err = r.removeChaosServices(engine, request)
```

Comment:
If the engine is global value then no need to pass in the function

Reply:
seems to be a little bit more understandable, if not causing issue with BCH, is this really a need?

Reply:
If it was a local variable then it will make sense to pass but for the global variable, you can get from any function and if you remove also BCH will not say anything

Reply:
@chandankumar4 It seems like we are already passing engine in some functions also, so it looks better to do by passing as an argument.

Reply:
Not some functions, but most of the fn's, it doesn't require that, so avoiding using glocal variable

Reply:
It's okay for now! So either we have to use global or make it local, Please create a respective issue for this

Reply:
This is good for `good first issue` maybe someone from community will pick cc @ksatchit

Reply:
cool

Reply:
Issue link: https://github.com/litmuschaos/litmus/issues/1209